bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/HasId.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.unifi.internal.api.model;
+
+/**
+ * Data classes that have an id as identifier.
+ *
+ * @author Hilbrand Bouwkamp - Initial contribution
+ */
+public interface HasId {
```

Comment:
These dto objects are exempt from null checks. Unfortunately the analysis tool specifically filter on dto only, and therefore these warnings remain.

Reply:
Ah. So I guess to avoid the warnings, three options:
- Rename to dto to comply with exemption in check.
- Add annotation
- Adapt analysis rule to support this naming also in exemptions.

Reply:
I've moved the classes to a `dto` package.